genny-project/layouts
# Intm 862

# Feature: https://outcomelife.atlassian.net/browse/INTM-862
depends on
https://outcomelife.atlassian.net/browse/INTM-338

# Description:
Added two buttons in timeline component to display questions for mid week and final week performance review.

# How To Test
1. Log in as an Agent.
2. Apply an intern to an internship
3. Move the application all the way to inprogress bucket.
4. Check the timeline component.
5. Click on 'Mid week review' -> this should display Mid week review question.
6. Click on 'Final week review' -> this should display Final week review question.
